primitives/src/authorization.rs

```diff
@@ -55,6 +55,9 @@ pub enum AuthorizationData<AccountId> {
     TransferCorporateActionAgent(Ticker),
     /// Authorization to become an agent of the `Ticker` with the `AgentGroup`.
     BecomeAgent(Ticker, AgentGroup),
+    /// Add Relayer paying key to user key
+    /// Must be issued by the paying key.
+    AddRelayerPayingKey(AccountId, AccountId),
```

Comment:
Would be good to clarify in the doc comment what the respective `AccountId`s represent.